<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BackgroundRemover</class>
 <widget class="QMainWindow" name="BackgroundRemover">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1091</width>
    <height>1047</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Multi-dataset fitting</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QSplitter" name="splitterBrowserDisplayy">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <layout class="QVBoxLayout" name="browserLayout">
        <item>
         <layout class="QHBoxLayout" name="actionsLayout">
          <item>
           <widget class="QPushButton" name="pushButtonSummonSliceSelector">
            <property name="toolTip">
             <string extracomment="Opens the Slice Selector"/>
            </property>
            <property name="text">
             <string>Load</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonFit">
            <property name="toolTip">
             <string>Start multi-dataset fitting</string>
            </property>
            <property name="text">
             <string>&amp;Fit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxPlotModel">
            <property name="text">
             <string>Plot Model</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonLoadNewSlices">
          <property name="text">
           <string>Load</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelFunctionBrowser">
          <property name="text">
           <string>FUNCTION BROWSER</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelFitOptionsBrowser">
          <property name="text">
           <string>FIT OPTIONS BROWSER</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QSplitter" name="splitterModelResiduals">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QHBoxLayout" name="horizontalLayoutModelDisplay">
         <item>
          <widget class="MantidQt::MantidWidgets::DisplayCurveFit" name="displayModelFit" native="true"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayoutModelDisplayOptions">
           <item>
            <widget class="QPushButton" name="pushButtonModelOption2">
             <property name="toolTip">
              <string>Add one or more spectra from a workspace</string>
             </property>
             <property name="text">
              <string>some action</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonModelOption1">
             <property name="toolTip">
              <string>Remove selected spectra</string>
             </property>
             <property name="text">
              <string>another action</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="modelDisplayOptionsSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget1">
        <layout class="QHBoxLayout" name="horizontalLayoutDisplayResiduals">
         <item>
          <widget class="MantidQt::MantidWidgets::PreviewPlot" name="displayResiduals" native="true"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayoutDisplayResidualsOptions">
           <item>
            <widget class="QPushButton" name="pushButtonDisplayResidualsOption2">
             <property name="toolTip">
              <string>Add one or more spectra from a workspace</string>
             </property>
             <property name="text">
              <string>some action</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonDisplayResidualsOption1">
             <property name="toolTip">
              <string>Remove selected spectra</string>
             </property>
             <property name="text">
              <string>another action</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacerDisplayResidualsOptions">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="layoutInfoBar">
      <item>
       <widget class="QPushButton" name="pushButtonHelp">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Open interface help page in your web browser.</string>
        </property>
        <property name="text">
         <string>?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="qlabelInfoBar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>infoBar</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacerInfoBar">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MantidQt::MantidWidgets::PreviewPlot</class>
   <extends>QWidget</extends>
   <header>MantidQtMantidWidgets/PreviewPlot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MantidQt::MantidWidgets::DisplayCurveFit</class>
   <extends>QWidget</extends>
   <header>MantidQtMantidWidgets/DisplayCurveFit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pushButtonModelOption1</tabstop>
  <tabstop>buttonFit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
